.gitignore

```diff
@@ -19,6 +19,7 @@ __pycache__/
 .Python
 build/
 _build/
+doc/build/
```

Comment:
Not 100% sure, but I thought that this was covered by https://github.com/open-sir/open-sir/pull/42/files#diff-a084b794bc0759e7a6b77810e01874f2R20